chore: remove flag from new dashboard modal

lharries opened the pull request:
## Changes

Removes the feature flag for the old new dashboard logic

## How did you test this code?

Ran it locally

lharries committed:
remove old new dashboard modal and feature flag

pauldambra reviewed (approved):
nice 🙌 

Didn't run it 🙈 but looks correct

lharries committed:
delete remaining action

github-actions[bot] committed:
Update UI snapshots for `chromium` (1)

github-actions[bot] committed:
Update UI snapshots for `webkit` (2)

github-actions[bot] committed:
Update UI snapshots for `webkit` (2)

lharries committed:
adds the OG template as a migration and fixes the tests

lharries committed:
Merge branch 'master' into luke/remove-new-dashboard-ff

github-actions[bot] committed:
Update UI snapshots for `chromium` (2)

lharries commented:
@pauldambra, the cypress tests failed with the new UI

I've updated them and in order to get the dashboard template test working I added the OG dashboard template as a migration

When/if the tests pass it would be great to get another review

github-actions[bot] committed:
Update UI snapshots for `chromium` (2)

lharries commented:
Tests seem to be passing

lharries commented:
Actually need to fix the remaining test briefly

lharries committed:
fix tests to account for new default dashboard template

lharries commented:
@pauldambra should be ready for a re-review now, if you wouldn't mind doubling checking the migration makes sense as I haven't added data to PostHog like this before

lharries commented:
And do I need to add a reverse step e.g. the Noop or is that optional?

lharries commented:
Seems like I've lost a commit which fixed the cypress tests, will add it back

lharries committed:
fixed cypress tests

pauldambra commented on posthog/migrations/0309_add_starter_dashboard_template.py:
```diff
@@ -0,0 +1,156 @@
+        ("posthog", "0308_add_indirect_person_override_constraints"),
+    ]
+
+    operations = [migrations.RunPython(create_starter_template)]
```
```suggestion
    operations = [migrations.RunPython(create_starter_template, reverse_code=migrations.RunPython.noop)]
```

RunPython throws when reversed if `reverse_code` is not provided. We _should_ be fine to have a no-op reversal instead

pauldambra reviewed (approved):
🚢

lharries committed:
Update posthog/migrations/0309_add_starter_dashboard_template.py

Co-authored-by: Paul D'Ambra <paul@posthog.com>

lharries committed:
fix manifest file

lharries committed:
merge

lharries committed:
Merge branch 'master' into luke/remove-new-dashboard-ff
